lib/rules/no-whitespace-before-property.js

```diff
@@ -16,6 +16,42 @@ module.exports = function(context) {
     var sourceCode = context.getSourceCode();
 
     //--------------------------------------------------------------------------
+    // Helpers
+    //--------------------------------------------------------------------------
+
+    /**
+     * Returns the name of the property.
+     * @param {Token} token - The token to check.
+     * @returns {string} The name of the node.
+     * @private
+     */
+    function getPropName(token) {
+        if (token.type === "Literal") {
+            return token.value;
```

Comment:
Ah, you're totally right. I'm going to try to implement something along the lines of what @mysticatea suggested.